packages/bvaughn-architecture-demo/src/suspense/PointsBoundingTimeCache.ts

```diff
@@ -0,0 +1,14 @@
+import { getPointsBoundingTimeResult } from "@replayio/protocol";
+
+import { ReplayClientInterface } from "shared/client/types";
+
+import { createGenericCache2 } from "./createGenericCache";
+
+export const {
+  getValueSuspense: getPointsBoundingTimeSuspense,
+  getValueAsync: getPointsBoundingTimeAsync,
+  getValueIfCached: getPointsBoundingTimeIfCached,
+} = createGenericCache2<ReplayClientInterface, [time: number], getPointsBoundingTimeResult>(
+  async (client, time) => client.getPointsBoundingTime(time),
+  time => `${time}`
+);
```

Comment:
This might be out of scope for these changes, but it would be great if we could make this cache a little more sophisticated. For instance:

If you ask for the points bounding `0:02` and the points you get are `point: 123, time: 0:01` and `point: 500, time: 0:05`, then those responses could be cached such that *any* time requested between those two points is going to return that same answer. I suspect this could really cut down on the number of requests we make.

Also, it's worth pointing out that [this PR](https://github.com/replayio/backend/pull/6733/files) is ready to go out, which will remove the `precise` flag, because it turns out it is not needed.